nifi-asymmetrik-standard-services/nifi-asymmetrik-standard-services-api-nar/pom.xml

```diff
@@ -46,7 +46,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.nifi</groupId>
-            <artifactId>nifi-standard-services-api-nar</artifactId>
```

Comment:
Are there other things using this library?

Reply:
You can only have one NAR parent per-nar-bundle. The AWS NAR bundle uses `nifi-standard-services-api-nar`as it's parent NAR, so it should allow any existing services to access anything from that they're already using, while adding access to use the AWSCredentialsProviderService.